onnxconverter_common/auto_mixed_precision_model_path.py

```diff
@@ -0,0 +1,250 @@
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+###########################################################################
+
+"""
+This tool converts a model to mixed precision (float32->float16) while excluding nodes as needed to maintain
```

Comment:
Suggest:
This tool converts a model to mixed precision (float32->float16) while excluding nodes as needed to maintain
a certain accuracy. After the conversion, the model will be saved on the disk under given path. A model with a size > 2G should leverage this.

Reply:
fixed